composites/OnboardingWizard/config/production-config.js

```diff
@@ -11,25 +11,25 @@ let configuration = {
 	},
 	"fields": {
 		"upsellConfigurationService": {
-			"componentName": "HTML",
+			"component": "HTML",
```

Comment:
I disagree with changing the name of this property. What is the reason to change it to `component`?